From 02730f2f28bdb5aef8e187e8e009b4e8fb7dc474 Mon Sep 17 00:00:00 2001
From: Kalyan Alle <kalyan.alle@amd.com>
Date: Tue, 24 Apr 2018 17:19:21 +0530
Subject: [PATCH 2600/4131] Compilation issues resolved v7

This patch resolves the compilation issues during porting
till commit : f4ab56b3dcc9a03062728e22ae4c8667c024732a
repo: brahma/ec/linux, branch: amd-18.10

Signed-off-by: kalyan alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/display/dc/os_types.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/os_types.h b/drivers/gpu/drm/amd/display/dc/os_types.h
index 3f9f9e4..86170b4 100644
--- a/drivers/gpu/drm/amd/display/dc/os_types.h
+++ b/drivers/gpu/drm/amd/display/dc/os_types.h
@@ -93,12 +93,8 @@
 	BREAK_TO_DEBUGGER(); \
 } while (0)
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 2, 0)
 #if defined(CONFIG_DRM_AMD_DC_DCN1_0)
 #include <asm/fpu/api.h>
 #endif
-#else
-#include <asm/i387.h>
-#endif
 
 #endif /* _OS_TYPES_H_ */
-- 
2.7.4

